docs/edgeql/funcops/datetime.rst

```diff
@@ -562,3 +562,27 @@ Date and Time
         {<duration>'1:20:45'}
         db> SELECT to_duration(seconds := 4845);
         {<duration>'1:20:45'}
+
+
+.. eql:function:: std::duration_to_seconds(cur: duration) -> float64
+
+    Return duration as total number of seconds in interval.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT duration_to_seconds(<duration>'1 hour');
+        {3600.0}
+        db> SELECT duration_to_seconds(<duration>'10 second 100 millis');
+        {10.1}
+
+
+.. eql:function:: std::duration_to_micros(cur: duration) -> int64
```

Comment:
Not sure this is needed, especially if we have a decimal cast. I think we should try to keep the number of std functions to the functional minimum and avoid adding trivial functions.

Reply:
Hm, the idea was that float can have precision issues. Decimal might work. But @elprans specifically asked for microseconds for some reason.